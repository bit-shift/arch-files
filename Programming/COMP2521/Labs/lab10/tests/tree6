12 90 8 7 10 13 15 16 
