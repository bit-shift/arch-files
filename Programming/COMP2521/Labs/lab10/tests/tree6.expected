Tree:
     12
     / \
    /   \
   /     \
  8      90
 / \     /
7  10   13
         \
         15
           \
           16

#nodes = 8,  #leaves = 3
Infix  : 7 8 10 12 13 15 16 90 
Prefix : 12 8 7 10 90 13 15 16 
Postfix: 7 10 8 16 15 13 90 12 
ByLevel: 12 8 90 7 10 13 15 16 
